spec/models/manageiq/providers/vmware/cloud_manager/refresher_spec.rb

```diff
@@ -134,6 +134,7 @@ def assert_specific_vm_powered_on
     )
 
     expect(v.ext_management_system).to eq(@ems)
+    expect(v.orchestration_stack.ems_ref)
```

Comment:
Sorry this was my fault, I said to expect the `ems_ref`, I meant `expect(v.orchestration_stack.ems_ref).to eq("vapp-bfd40328-a383-4b0e-9130-66edd590f4ab")` or now that you have `@orchestration_stack2` you can `expect(v.orchestration_stack).to eq(@orchestration_stack2)`